extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -578,9 +582,7 @@ export class AmpIframe extends AMP.BaseElement {
    * @private
    */
   looksLikeTrackingIframe_() {
-    const box = this.element.getLayoutBox();
-    // This heuristic is subject to change.
-    if (box.width > 10 && box.height > 10) {
+    if (!looksLikeTrackingIframe(this.element.getLayoutBox())) {
```

Comment:
Might be worth moving the remaining isInContainer logic to the helper as well remove the private `looksLikeTrackingIframe_` specially given you would need to duplicate that logic for video one anyway.